corehq/apps/hqcase/utils.py

```diff
@@ -272,7 +272,7 @@ def update_case(domain, case_id, case_properties=None, close=False):
     context = {
         'case_id': case_id,
         'date_modified': json_format_datetime(datetime.datetime.utcnow()),
-        'user_id': '',
+        'user_id': 'system',
```

Comment:
updating `submit_case_blocks` also sounds reasonable, though I think we should do that separately and be rather thoughtful/careful since it is called in a large number of places.